lib/jwt_sessions.rb

```diff
@@ -89,8 +91,22 @@ def algorithm
     @algorithm ||= DEFAULT_ALGORITHM
   end
 
+  def token_store=(args)
+    @token_store =
+      if args.is_a?(JWTSessions::StoreAdapters::AbstractStoreAdapter)
+        args
+      elsif args.is_a?(Array)
+        JWTSessions::StoreAdapters::AbstractStoreAdapter.instance_by_symbol(args.first, args.last)
+      end
+  end
+
   def token_store
-    RedisTokenStore.instance(redis_url, token_prefix)
+    @token_store ||=
+      if ENV['STORE_ADAPTER'].nil?
```

Comment:
As we're setting the store through ENV for the tests sake only, it'd make sense to set the store in test helpers (or setups) and not in the core code.

Reply:
agreed